playwright/pyproject.toml

```diff
@@ -0,0 +1,5 @@
+# pyproject.toml
+[tool.pytest.ini_options]
+filterwarnings = [
+    'ignore::DeprecationWarning',
```

Comment:
pytest-playwright uses the https://pyee.readthedocs.io/en/latest/index.html library for managing some of it's async and async-adjacent aspects. We should revisit this periodically to see if we can get rid of this after an update.